src/StackHCI/StackHCIVm/README.md

```diff
@@ -0,0 +1,666 @@
+<!-- region Generated -->
+# Az.StackHCIVm
+This directory contains the PowerShell module for the StackHciVM service.
+
+---
+## Status
+[![Az.StackHCIVm](https://img.shields.io/powershellgallery/v/Az.StackHCIVm.svg?style=flat-square&label=Az.StackHCIVm "Az.StackHCIVm")](https://www.powershellgallery.com/packages/Az.StackHCIVm/)
+
+## Info
+- Modifiable: yes
+- Generated: all
+- Committed: yes
+- Packaged: yes
+
+---
+## Detail
+This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 2.2.3 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.StackHCIVm`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+---
+
+## Generation Requirements
+
+Use of the beta version of `autorest.powershell` generator requires the following:
+
+- [NodeJS LTS](https://nodejs.org) (10.15.x LTS preferred)
+  - **Note**: It _will not work_ with Node < 10.x. Using 11.x builds may cause issues as they may introduce instability or breaking changes.
+    > If you want an easy way to install and update Node, [NVS - Node Version Switcher](../nodejs/installing-via-nvs.md) or [NVM - Node Version Manager](../nodejs/installing-via-nvm.md) is recommended.
+- [AutoRest](https://aka.ms/autorest) v3 beta <br>`npm install -g autorest@autorest`<br>&nbsp;
+- PowerShell 6.0 or greater
+  - If you don't have it installed, you can use the cross-platform npm package <br>`npm install -g pwsh`<br>&nbsp;
+- .NET Core SDK 2.0 or greater
+  - If you don't have it installed, you can use the cross-platform npm package <br>`npm install -g dotnet-sdk-2.2`<br>&nbsp;
+
+## Run Generation
+
+In this directory, run AutoRest:
+
+> `autorest-beta`
+
+---
+
+### AutoRest Configuration
+
+> see https://aka.ms/autorest
+
+```yaml
+require:
+  - $(this-folder)/../../readme.azure.noprofile.md
+repo: azure-rest-api-specs-pr
```

Comment:
use public repo

Reply:
even with the public repo the autogenerated files are the same, they are generated in this format StackHciVM

Reply:
<img width="483" alt="image" src="https://github.com/Azure/azure-powershell/assets/43424696/6f42e14b-e4b0-4c3a-a363-26da29ea8f38">